test/analytics.test.js

```diff
@@ -111,6 +111,21 @@ describe('Analytics', function() {
       group.load.restore();
     });
 
+    it('should gracefully handle integrations that fail to initialize', function() {
+      Test.prototype.initialize = function() { throw new Error('Uh oh!'); };
+      analytics.add(Test);
+      analytics.initialize();
+      assert(analytics.initialized);
+    });
+
+    it('should store the names of integrations that did not initialize', function() {
```

Comment:
Could we add a test for the case where integration.prototype and/or integration.prototype.name is undefined?

Reply:
Yeah good call.